cassandra-bigtable-proxy/compliance/schema_setup/cassandra_schema_setup.go

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package schema_setup
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/gocql/gocql"
+	"github.com/ollionorg/cassandra-to-bigtable-proxy/compliance/utility"
+)
+
+// Column represents the definition of a single column in the table schema
+type Column struct {
+	ColumnName   string `json:"ColumnName"`
+	ColumnType   string `json:"ColumnType"`
+	IsPrimaryKey string `json:"IsPrimaryKey"`
+	PKPrecedence string `json:"PK_Precedence"`
+}
+
+// Table represents the schema definition for a single table
+type Table struct {
+	TableName string   `json:"table_name"`
+	Columns   []Column `json:"columns"`
+}
+
+// CassandraTableSchema represents the schema for multiple Cassandra tables
+type CassandraTableSchema struct {
+	Keyspace string  `json:"keyspace"`
+	Tables   []Table `json:"tables"`
+}
+
+// SetupCassandraSchema sets up the Cassandra keyspace and tables
+func SetupCassandraSchema(session *gocql.Session, schemaFilePath, keyspace string) error {
```

Comment:
what is the relationship between this keyspace and the one taht is part of CassandraTableSchema?